[2018-04-11T02:26:11.482044] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] Initiating pyFlow run
[2018-04-11T02:26:11.507394] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] pyFlowClientWorkflowClass: Runner
[2018-04-11T02:26:11.565653] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] pyFlowVersion: 1.1.17
[2018-04-11T02:26:11.590737] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] pythonVersion: 2.7.13.final.0
[2018-04-11T02:26:11.615831] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] WorkingDir: '/home/gumm'
[2018-04-11T02:26:11.640894] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] ProcessCmdLine: '/nfs/turbo/lsa-duhaimem/devel/gumm/workflows/arg_identifier/runner.py /nfs/turbo/lsa-duhaimem/usda_med_umich/analysis/Run_1768/02_ASSEMBLY/full-co-assembly/final.contigs.fa -r ./dependencies/data/card/model/nucleotide_fasta_protein_homolog_model.fasta -o /nfs/turbo/lsa-duhaimem/usda_med_umich/analysis/Run_1768/13_ARGS_IN_CONTIGS -a duhaimem_fluxm -p 12 -m 300000 -w 2:00:00 --flux --no-dispatch'
[2018-04-11T02:26:11.665993] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [RunParameters] mode: flux
[2018-04-11T02:26:11.691083] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [RunParameters] nCores: 12
[2018-04-11T02:26:11.716171] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [RunParameters] memMb: unlimited
[2018-04-11T02:26:11.741558] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [RunParameters] dataDir: /nfs/turbo/lsa-duhaimem/usda_med_umich/analysis/Run_1768/13_ARGS_IN_CONTIGS/logs/pyflow.data
[2018-04-11T02:26:11.766651] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [RunParameters] isDryRun: False
[2018-04-11T02:26:11.791734] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [RunParameters] isContinue: False
[2018-04-11T02:26:11.808473] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [RunParameters] isForceContinue: False
[2018-04-11T02:26:11.825237] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [RunParameters] mailTo: ''
[2018-04-11T02:26:11.842487] [nyx7010.arc-ts.umich.edu] [8643_1] [TaskRunner:masterWorkflow] Starting task specification for master workflow
[2018-04-11T02:26:11.873794] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR] Failed to complete master workflow, error code: 1
[2018-04-11T02:26:11.892165] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR] errorMessage:
[2018-04-11T02:26:11.908925] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR] Unhandled Exception in TaskRunner-Thread-masterWorkflow
[2018-04-11T02:26:11.925668] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR] Traceback (most recent call last):
[2018-04-11T02:26:11.942432] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR]   File "/nfs/turbo/lsa-duhaimem/devel/gumm/workflows/arg_identifier/dependencies/miniconda/lib/python2.7/site-packages/pyflow/pyflow.py", line 1069, in run
[2018-04-11T02:26:11.959194] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR]     (retval, retmsg) = self._run()
[2018-04-11T02:26:11.975931] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR]   File "/nfs/turbo/lsa-duhaimem/devel/gumm/workflows/arg_identifier/dependencies/miniconda/lib/python2.7/site-packages/pyflow/pyflow.py", line 1121, in _run
[2018-04-11T02:26:11.992701] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR]     self.workflow.workflow()
[2018-04-11T02:26:12.009459] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR]   File "/nfs/turbo/lsa-duhaimem/devel/gumm/workflows/arg_identifier/runner.py", line 26, in workflow
[2018-04-11T02:26:12.026199] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR]     self.contigs_fasta, self.reference, pblat_output))
[2018-04-11T02:26:12.042965] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR]   File "/nfs/turbo/lsa-duhaimem/devel/gumm/workflows/arg_identifier/dependencies/miniconda/lib/python2.7/site-packages/pyflow/pyflow.py", line 3505, in addTask
[2018-04-11T02:26:12.059727] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR]     memMb = int(memMb)
[2018-04-11T02:26:12.076562] [nyx7010.arc-ts.umich.edu] [8643_1] [WorkflowRunner] [ERROR] ValueError: invalid literal for int() with base 10: 'unlimited'
